controllers/install/eventingester_controller.go

```diff
@@ -33,15 +37,20 @@ import (
 	"github.com/armadaproject/armada-operator/controllers/builders"
 )
 
+const (
+	eventIngesterVolumeConfigKey = "user-config"
+	eventIngesterFinalizer       = "batch.tutorial.kubebuilder.io/finalizer"
```

Comment:
We probably should change the finalizer name to be related to our project.  I noticed it in Dejan's PR also.  

```suggestion
	eventIngesterFinalizer       = "armadaproject.armadaoperator/finalizer"
```

Reply:
should the nane also indicate which component, eg "eventingester-finalizer"? I don't have need for this at the time so I can just remove it too

Reply:
I'm not sure.  They are mainly used to say 'don't delete yet'.  So I figured we can add a general one for our components

Reply:
i just removed finalizer from this component since OwnerReference should handle the dependent items